docs/adr/adr-006-email-delivery-responsibilities.md

```diff
@@ -0,0 +1,110 @@
+# 6. Email Delivery Responsibilities
+
+Date: 2020-10-23
+
+## Context
+
+Email, as an asynchronous communication medium, inherently has two boolean
+scenarios that need to be true for an email to be received. An email has to be
+_sent_ to a mail server and the mail server then needs to _deliver_ the email to the
+recipient's mail server. Resolving whether an email is sent is a rather simple
+and question that can be resolved synchronously - "did the mail server accept the
+email?" - however, determining whether an email is received is more complex
+and it can be a drawn out process as mail servers may retry delivery over a
+period of time when encountering failures. Typically, in the medium of email,
+clients only consider the first scenario in reporting an email's status, this
+is the point an email becomes sent. Whereas the latter scenario isn't typically
+reflected in email clients, it is normal to assume the email was received
+successfully unless later you receive an email indicating a bounce has occurred.
+
+The [initial Email Alert API Notify integration][adr-1] was designed to
+consider both the sending and delivery of email. It had systems to monitor
+whether Notify had managed to deliver an email to a recipient and an automatic
+retry mechanism for when Notify had problems delivering an email.
+
+In September 2020 we decided to re-evaluate this concept conflation as we were
+concerned that Notify functionality was duplicated in Email Alert API and
+that this had an adverse effect on the complexity of the system.
+
+In September 2020, we re-evaluated the need for this level of functionality and
+concluded that this was unnecessary as it duplicated features that
+Notify provided. We chose to simplify the system and re-define the
+responsibilities of email delivery. We decided that Email Alert API
```

Comment:
```suggestion
responsibilities of email delivery.
```

Minor: we repeat this decision in the next paragraph (in the decision section), so we don't need to say it here.